wercker-step.yml

```diff
@@ -1,16 +1,20 @@
 name: runscope-bucket-trigger
-version: 1.0.1
+version: 1.0.10
```

Comment:
I had an older wercker-step with the same name, but deleted it.
Ever since I recreated this one, the `Details` page doesn't show up.

I am hoping that it will be displayed again, after a version is display, which was not there in the old step.